src/main/java/in/ashwanthkumar/gocd/github/GitHubPRBuildPlugin.java

```diff
@@ -199,10 +208,13 @@ GoPluginApiResponse handleLatestRevisionSince(GoPluginApiRequest goPluginApiRequ
                     String latestSHA = newerRevisions.get(branch);
                     Revision revision = git.getDetailsForRevision(latestSHA);
 
-                    Map<String, Object> revisionMap = getRevisionMap(gitConfig, branch, revision, newBranchToRevisionMap);
+                    Map<String, Object> revisionMap = getRevisionMap(gitConfig, branch, revision);
                     revisions.add(revisionMap);
                 }
                 response.put("revisions", revisions);
+                Map<String, String> scmDataMap = new HashMap<String, String>();
+                scmDataMap.put(BRANCH_TO_REVISION_MAP, JSONUtils.toJSON(newBranchToRevisionMap));
+                response.put("scm-data", scmDataMap);
```

Comment:
yes. i added the ability to set & get `scm-data` in this PR - https://github.com/gocd/gocd/pull/952. Its in & an installer is out so this can be merged now.